src/components/PostInfo/PostInfo.jsx

```diff
@@ -1,3 +1,32 @@
-export const PostInfo = () => (
-  <>Put the post here</>
-);
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post }) => {
+  const { title, user, body, comments } = post;
```

Comment:
const { 
 title, 
 user, 
 body, 
 comments } = post;
 
 it would be better